### DISCUSSION TITLE
how to get webhook custom params?

### DISCUSSION THREAD

**Discussion by dengao123** [minio/minio#17471] [Q&A] [answered]
the webhook enpoint is ok, but cannot get custom params

**Comment by harshavardhana** [comment]
?

**Reply by dengao123** [reply]
I added customized parameters when uploading. I want to get the corresponding parameters in the webhook triggered after uploading, similar to Alibaba Cloud's [callback](https://help.aliyun.com/document_detail/31989.htm?spm=a2c4g.31853.0.0.51e13781YS7AGQ#reference-zkm-311-hgb)

**Reply by harshavardhana** [reply, marked as answer]
We do not implement anything related to Alibaba. We are S3 only.

**Reply by dengao123** [reply]
so，what should I do if I want to achieve the same functionality

**Reply by harshavardhana** [reply]
Use Alibaba Cloud @dengao123
